bluebottle/suggestions/views.py

```diff
@@ -1,14 +1,14 @@
 from datetime import date
 from rest_framework import generics, status, response
-from rest_framework.permissions import IsAuthenticated
+from rest_framework.permissions import AllowAny
 
 from bluebottle.suggestions.models import Suggestion
 from bluebottle.suggestions.serializers import SuggestionSerializer
 
 
 class SuggestionList(generics.ListCreateAPIView):
     queryset = Suggestion.objects.all()
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (AllowAny,)
     serializer_class = SuggestionSerializer
```

Comment:
Same here, not anybody should be able to read the list of suggestions. Anybody should be able to create them though.